src/components/organisms/AppRouter/AppRouter.jsx

```diff
@@ -60,6 +62,12 @@ const AppRouter = () => {
         <Route path="/admin/venue/creation" component={VenueWizard} />
         <Route path="/admin/venue/edit/:venueId" component={VenueWizard} />
 
+        <Route path="/admin_v2/venue/:venueId?" component={AdminVenueView} />
+        <Route
+          path="/admin_v2/advanced-settings/:venueId?"
```

Comment:
Why is path using both underscore and dash? Is there an obstacle of sticking to one?

Reply:
This was introduced this way by an ex-dev and he decided to use the underscore syntax. We just went with it. It's just a temp url to test the new functionality. Our final goal is to make this `/admin`

Reply:
The _ is legacy and will go, the - is likely canon and will stay.

Reply:
I will change the routes, Its not a big deal.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1511/commits/0d4d60cd9f66fb4682d6f3cbf71193467e5c5a52

Reply:
The names of the components and imports need to be changed too, but I don't want to make this PR too big, because we should merge this asap. I will make sure to check if there is an issue for this, if there is not will create one.

Reply:
Backlogged here: https://github.com/sparkletown/internal-sparkle-issues/issues/807